packages/create-schemas/tests/plugins.test.ts

```diff
@@ -69,4 +70,33 @@ describe.concurrent("plugins", () => {
           "
         `);
     });
+
+    test("openapiMSWPlugin", async ({ expect }) => {
```

Comment:
Question: Do we have any guideline for how we write test cases? It is usually a good practice to write test scenarios in a sentence.(e.g. [an example](https://github.com/Shopify/web-configs/blob/main/packages/babel-preset/tests/babel-preset.test.ts#L24)) For example: `openapiMSWPlugin generates right files when it is configured`.

Reply:
We don't have a convention for test names. So far, I've favored shorter names, but I don't mind using long names, especially as we add more test cases.